code/game/mecha/working/ripley.dm

```diff
@@ -54,22 +55,31 @@
 
 /obj/mecha/working/ripley/update_desc()
 	. = ..()
-	if(!hides) // Just in case if hides are somehow removed
+	if(!hides && !plates) // Just in case if armour are somehow removed
 		desc = initial(desc)
 		return
 	if(hides == 3)
 		desc = "Autonomous Power Loader Unit. It's wearing a fearsome carapace entirely composed of goliath hide plates - its pilot must be an experienced monster hunter."
 	else
-		desc = "Autonomous Power Loader Unit. Its armour is enhanced with some goliath hide plates."
+		desc = "Autonomous Power Loader Unit. It's armour is enhanced with some goliath hide plates."
+	if(plates == 3)
+		desc = "Autonomous Power Loader Unit. It's armour is completely lined with metal inserts and pieces of debris - the owner of this mecha clearly not skimped on safety."
+	else
+		desc = "Autonomous Power Loader Unit. It's armor is reinforced with metal inserts"
+
```

Comment:
```suggestion
		desc = "Autonomous Power Loader Unit. Its armour is enhanced with some goliath hide plates."
	if(plates == 3)
		desc = "Autonomous Power Loader Unit. Its armour is completely lined with metal plating."
	else
		desc = "Autonomous Power Loader Unit. It's armor is reinforced with some metal plating."
```
I'd hesitate at calling it `debris`, as you are professionally producing it in Robotics.